sct.py

```diff
@@ -1021,44 +1021,71 @@ def prepare_aws_region(region):
     aws_region.configure()
 
 
-@cli.command("create-runner-image", help="Create an SCT runner image in selected AWS region. "
+@cli.command("create-runner-image", help="Create an SCT runner image in selected AWS or GCE region. "
                                          f"If the requested region is not a source region "
-                                         f"({SctRunner.SOURCE_IMAGE_REGION}) the image will be first created in the"
+                                         f"(aws: {AwsSctRunner.SOURCE_IMAGE_REGION}, gce: {GceSctRunner.SOURCE_IMAGE_REGION})"
+                                         f" the image will be first created in the"
                                          f" source region and then copied to the chosen one.")
-@click.option("-r", "--region", required=True, type=click.Choice(all_aws_regions(cached=True)),
+@click.option("-c", "--cloud-provider", required=True, type=str, default="aws",
```

Comment:
type=click.Choice('aws', 'gce')

Reply:
fixed to `type=click.Choice(['aws', 'gce'])`
Thanks.